ui/tests/acceptance/job-status-panel-test.js

```diff
@@ -819,6 +821,7 @@ module('Acceptance | job status panel', function (hooks) {
           lost: 0,
           complete: 0.2,
         },
+        groupsCount: 1,
```

Comment:
Test was flaky without this because sometimes it would create 2 tasks groups, so the total number of allocations doubled.

Reply:
Extracted to https://github.com/hashicorp/nomad/pull/17648